index.html

```diff
@@ -10,346 +10,308 @@
     <meta name="description" content="mobile first web design" />
     <link rel="icon" type="image/png" sizes="32x32" href="images/logo.png" />
     <link rel="stylesheet" href="css/styles.css" />
-    <link
-      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-    <script
-      src="https://kit.fontawesome.com/6d267a36ba.js"
-      crossorigin="anonymous"
-    ></script>
-    <style>
-      li {
-        display: inline;
-        padding: 5px;
-      }
-    </style>
   </head>
   <body class="main">
     <!-- navbar setup -->
     <header>
+      <!-- phone navigation -->
+      <section class="phone-navigation">
+        <a href="/" class="logo">
+          <h3>Welcome</h3>
+        </a>
+        <button type="button" class="menu-button">
+          <img src="../assets/Union.svg" alt="menu link" />
+        </button>
+        <nav class="mobile-navigation hide">
+          <button type="button" class="close-button">
+            <img src="../assets/Icon-Cancel.svg" alt="cancel icon" />
+          </button>
+          <ul>
+            <li>
+              <a href="#" class="port-link"> Portfolio </a>
+            </li>
+            <li>
+              <a href="#" class="abt-link"> </a>
+            </li>
+            <li>
+              <a href="#" class="contact-link"> Contact </a>
+            </li>
+          </ul>
+        </nav>
+      </section>
+    </header>
+    <!-- home page -->
+    <section class="home-page">
       <nav>
-        <ul class="menu">
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
+        <ul class="desktop-navigation">
+          <li><a href="#recent-works">Portfolio</a></li>
+          <li><a href="#about-me">About</li>
+          <li><a href="#about-me">Contact</li>
```

Comment:
- **[OPTIONAL]** Well done adding the `href` to the links 👏🏽 but there is a mistake on the Contact link the `href `should go to the contact form section.